src/components/Navbar.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import axios from 'axios';
+import { useSelector } from 'react-redux';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Button from '@material-ui/core/Button';
+import { makeStyles } from '@material-ui/core/styles';
+import AppStyles from '../assets/jss/styles/App.styles.jsx'
+import SVG from 'react-inlinesvg';
+import { API_HOST } from '../constants'
+import { Redirect } from 'react-router-dom';
+
+const useStyles = makeStyles(AppStyles);
+
+function logout() {
+  axios({ method: 'DELETE', url: `${API_HOST}/api/auth`,
+                headers: { Authentication: "Bearer " +  window.localStorage.getItem('authToken')} })
```

Comment:
We should add it to axios configuration. It will set this header by default (If I'm not mistaken I have already add it)

Reply:
Check App.jsx, setAxiosInterceptors() function, just add Bearer prefix to it

Reply:
Got you, thanks

Reply:
nevermind, I fixed this already :)